verification/io/router.gobra

```diff
@@ -20,6 +20,10 @@ package io
 
 // TODO: make individual_router an interface type with the following methods
 
+ghost
+decreases
+pure func core_as_set() set[IO_as]
```

Comment:
this function is called `core` in the original io-spec. It is fine by me to keep the nameyou picked, but we should probably add a comment here. Also, can you please add a comment explaining the purpose of this function?

Reply:
Can you please address this comment as well?